### PR TITLE
Explicit paddlepaddle installation in the demo steps

### PR DESCRIPTION
**Pull request by mzegla**

### CODE REVIEW

**Review by dtrawins — APPROVED** [review]

**Review by dkalinowski — APPROVED** [review]

File: demos/segmentation_using_paddlepaddle_model/python/README.md [hunk 1]
```diff
@@ -20,6 +18,13 @@ git clone https://github.com/openvinotoolkit/model_server.git
 cd model_server/demos/segmentation_using_paddlepaddle_model/python
 ```
 
+Install [paddlepaddle](https://pypi.org/project/paddlepaddle/) and other packages required to export the model to deployable format
+```bash
+pip3 install paddlepaddle
+pip3 install pyyaml
+pip3 install scipy
```
Comment (jasiu86) [inline demos/segmentation_using_paddlepaddle_model/python/README.md:25]: Can we use venv mechanism like in many other examples?

image

Comment (jasiu86) [reply demos/segmentation_using_paddlepaddle_model/python/README.md:25]: image

Comment (dtrawins) [reply demos/segmentation_using_paddlepaddle_model/python/README.md:25]: we are not enforcing the use of virtual env in other samples, that would be out of scope for this PR

Comment (ngrozae) [reply demos/segmentation_using_paddlepaddle_model/python/README.md:25]: can we enclose it in requirements file and add versions to it? this way we could be sure that there would be no problems     with wrong versions in the future

**Pull request merged**